src/main.rs

```diff
@@ -153,24 +166,24 @@ async fn start_tauri() {
 #[cfg(feature = "ut")]
 async fn start_ut() {
     use std::process::Command;
-    use std::process::{exit, Stdio};
+    use std::process::Stdio;
 
     log::info!("Starting the ubuntu touch client");
-    thread::spawn(|| {
-        let rt = tokio::runtime::Runtime::new().unwrap();
-        rt.block_on(async {
-            let route = warp::fs::dir("./axolotl-web/dist");
-            warp::serve(route).run(([127, 0, 0, 1], 9081)).await;
-        });
+    // Why do we start warp another time?
```

Comment:
Why do we start warp another time?

Reply:
if we use ubuntu touch, the ui html/js has to be served via a file server. In tauri, the code is bundled with tauri. Actually we need another file server for the attachments.
It's not necessary to start another server, but we could add it as an route to warp if the feature ut is used.

Reply:
understood